docs/IntegrationWithExistingApps.md

```diff
@@ -573,8 +573,12 @@ AppRegistry.registerComponent('HelloWorld', () => HelloWorld);
 ## Prepare your current app
 
 In your app's `build.gradle` file add the React Native dependency:
-
-    compile "com.facebook.react:react-native:+"  // From node_modules
+```
+dependencies {
+    ...
+    compile "com.facebook.react:react-native:+" // From node_modules. Replace + with actual React Native version.
```

Comment:
@jankalfus

> It is recommended by Android Studio to always define concrete versions of packages

Yes, but we already have the version defined in `package.json`

> This ensures compatibility. For example, if you use "com.facebook.react:react-native:0.30.0", yet you've downloaded React Native v0.31.0 into the node_modules folder, Gradle sync won't finish successfully and thus Android Studio will notify you of the issue.

It's a good point. But in my experience it has caused lots of confusion during upgrade. Now, to update, you've to change the version number at 2 places. If you forget to update one place, the build will fail, but you might not know why, because android studio won't tell you that the version number doesn't match with the version in package.json.